### PR TITLE
assert len(result) compared with 0 instead of 110

### PR DESCRIPTION
**Pull request by aryanbhosale**
# Pull Request

## Description

_Please delete the italicised instruction text!
Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context. List any dependencies that are required for this change._

Fixes #

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration_

- [ ] Yes

_If your changes affect data processing, have you plotted any changes? i.e. have you done a quick sanity check?_

- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/india_api/internal/inputs/indiadb/test_indiadb.py [hunk 1]
```diff
@@ -22,15 +22,15 @@ def test_get_predicted_wind_yields_for_location(self, client, forecast_values) -
         locID = "testID"
         result = client.get_predicted_wind_yields_for_location(locID)
 
-        assert len(result) == 110
+        assert len(result) > 0
```
Comment (peterdudfield) [inline src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: im very suprised this number changes? what does it change too? 

What timezone are you in? Just incase that is causing the problems?
Did you not have any errors with this test

Comment (aryanbhosale) [reply src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: > im very suprised this number changes? what does it change too? 
> 
> What timezone are you in? Just incase that is causing the problems?
> Did you not have any errors with this test

It changes to 47. my timezone is IST(UTC+5.30) - should I work around with that in the code?
When I changed this from ==110 to >0 I didn't have issues, all 14 tests passed

Comment (aryanbhosale) [reply src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: > im very suprised this number changes? what does it change too? 
> 
> What timezone are you in? Just incase that is causing the problems?
> Did you not have any errors with this test

Should I be changing the "testId" to "IST" instead ?

Comment (peterdudfield) [reply src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: hmm, im not sure

Im reallu suprised its 47. If you looked here, https://github.com/openclimatefix/india-api/blob/main/src/india_api/internal/inputs/indiadb/conftest.py#L118, there are 110 forecast values made. If you remove your `int` change does it change that?

Comment (aryanbhosale) [reply src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: > hmm, im not sure
> 
> Im reallu suprised its 47. If you looked here, https://github.com/openclimatefix/india-api/blob/main/src/india_api/internal/inputs/indiadb/conftest.py#L118, there are 110 forecast values made. If you remove your `int` change does it change that?

I tried it without typecasting powerkw to int, it still shows 47, is this expected behaviour?

Comment (peterdudfield) [reply src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: weird. That is odd. No it should be 110. Thats what the current `main` branch does. From the 47, can you see what tmestamps they contain, and what systems they are from?

Comment (aryanbhosale) [reply src/india_api/internal/inputs/indiadb/test_indiadb.py:25]: > weird. That is odd. No it should be 110. Thats what the current `main` branch does. From the 47, can you see what tmestamps they contain, and what systems they are from?

I deleted my og repo and reforked it from the current ```main``` branch and retried. It gives the same result(47) with the timestamps(I printed ```result```):
image

**Comment by aryanbhosale** [comment]
I've noticed another interesting thing. Initially my process was :
1. Clone
2. Navigate to india-api
3. Create venv and activate it
4. pip install -e .
5. pip install pytest and testcontainers
6. pytest(this resulted in failure of 2 cases)
========================================================
Now when I try to do pip install -e ".[all]" -> it installs a few more devdependencies
Then when i run pytest again, it stops(or rather never stops) here ==> It's an infinite loop:
image

**Comment by peterdudfield** [comment]
> I've noticed another interesting thing. Initially my process was :
> 
> 1. Clone
> 2. Navigate to india-api
> 3. Create venv and activate it
> 4. pip install -e .
> 5. pip install pytest and testcontainers
> 6. pytest(this resulted in failure of 2 cases)
>    ========================================================
>    Now when I try to do pip install -e ".[all]" -> it installs a few more devdependencies
>    Then when i run pytest again, it stops(or rather never stops) here ==> It's an infinite loop:
>    image

ah, i think that might be a bug in testcontainers, good to pin this back to <4.0. I might be that, it might not be

**Comment by aryanbhosale** [comment]
> > I've noticed another interesting thing. Initially my process was :
> > 
> > 1. Clone
> > 2. Navigate to india-api
> > 3. Create venv and activate it
> > 4. pip install -e .
> > 5. pip install pytest and testcontainers
> > 6. # pytest(this resulted in failure of 2 cases)
> >    Now when I try to do pip install -e ".[all]" -> it installs a few more devdependencies
> >    Then when i run pytest again, it stops(or rather never stops) here ==> It's an infinite loop:
> >    image
> 
> ah, i think that might be a bug in testcontainers, good to pin this back to <4.0. I might be that, it might not be

you mean roll testcontainers back to <4.0? Isn't it already 3.7.1??

**Comment by aryanbhosale** [comment]
All tests pass finally! My inference from this exercise is that the tests pass on VMs, WSL, and Linux systems. And that all the forecasts fail to quantify on windows systems
image

**Comment by aryanbhosale** [comment]
> > All tests pass finally! My inference from this exercise is that the tests pass on VMs, WSL, and Linux systems. And that all the forecasts fail to quantify on windows systems image
> 
> I'm attaching the tests and opening a new PR for this

I've done this in #34

**Pull request closed without merging**